### DISCUSSION TITLE
Bun v1.1.26

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#13503] [Announcements]
To install:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.26:
```bash
bun upgrade
```

### Thanks to 3 contributors!

- [@cirospaciari](https://github.com/cirospaciari)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.26'>Bun v1.1.26</a>.</em>

**Comment by ZeinBarhoum** [comment]
Please add `bun-linux-x64.zip` to the assets. Now the script `https://bun.sh/install` is broken on linux

**Reply by Jarred-Sumner** [reply]
Fixed.
